modules/monitoring-creating-alert-routing-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-alerts.adoc
+
+:_content-type: PROCEDURE
+[id="creating-alert-routing-for-user-defined-projects_{context}"]
+= Creating alert routing for user-defined projects
+ 
+[IMPORTANT]
+====
+[subs="attributes+"]
+Alert routing for user-defined projects is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview[https://access.redhat.com/support/offerings/techpreview].
+====
+
+[role="_abstract"]
+You can create alerting routing for user-defined projects. 
+
+.Prerequisites
+
+* An administrator has enabled monitoring for user-defined projects.
+* You are logged in as a user that has the `alert-routing-edit` role for the project where you want to create alert routing.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Create a YAML file for alert routing. In this example, it is called `example-app-alert-routing.yaml`.
+
+. Add an `AlertmanagerConfig` YAML definition to the file. For example:
++
+[source,yaml]
+----
+apiVersion: monitoring.coreos.com/v1alpha1
+kind: AlertmanagerConfig
+metadata:
+  name: example-routing
+  namespace: ns1
+spec:
+  route:
+    receiver: default
+    groupBy: [job]
+  receivers:
+  - name: default
+    webhookConfigs:
+    - url: https://httpbin.org/post
+----
++
```

Comment:
Maybe we need a note saying that the AlertmanagerConfig object only applies to alerts from PrometheusRules in the same namespace?

Reply:
IIUC that was covered in the text: 

```
For user-defined alerting rules, user-defined routing is scoped to the namespace in which the resource is defined.
For example, a routing configuration in namespace `ns1` only applies to `PrometheusRules` resources in the same namespace.
```

Reply:
Right but I think that it's worth reiterating here since the audiences might be different (cluster admin vs. user)?

Reply:
Yes. I agree it's worth repeating here for the reasons Simon points out. I'll put in a note.

Reply:
@simonpasquier I added a note. PTAL. Thanks!